README.md

```diff
@@ -14,7 +14,7 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 ### Installation
 
-+ Install [Homebrew](http://brew.sh/) (OSX):
+#### Mac OSX: Install [Homebrew](http://brew.sh/):
```

Comment:
I would suggest make 2 lines.

Reply:
When you say make two lines, you are referring to the section breaks/horizontal rules?